.circleci/config.yml

```diff
@@ -540,6 +583,23 @@ jobs:
           enable-dashboard: << parameters.enable-dashboard >>
           suite: << parameters.suite >>
 
+  #Maybe have a move this to a image folder
+  docker-deploy:
```

Comment:
This job name should properly describe what the job does. The job doesn't deploy anything to docker, it builds the builder image. Deploying to docker makes me think you are launching a container

Reply:
Changed the name.